bigbluebutton-html5/imports/ui/components/common/button/styles.js

```diff
@@ -175,11 +200,83 @@ const ButtonWrapper = styled(BaseButton)`
     padding: ${jumboPaddingY} ${jumboPaddingX};
   `}
 
-  & > span {
-    display: block;
-    text-align: center;
-    white-space: nowrap;
-  }
+  ${({ color }) => color === 'primary' && `
+    &:focus:not([aria-disabled="true"]){
+      & > span{
+        color: ${btnPrimaryColor};
+        background-color: ${btnPrimaryBg};
+        background-clip: padding-box;
+        box-shadow: 0 0 0 ${borderSize} ${btnPrimaryBorder};
+      }
+    }
+
+    &:hover{
+      & > span{
+        filter: brightness(90%);
+        color: ${btnPrimaryColor};
+        background-color: ${btnPrimaryHoverBg} !important;
+      }
+    }
+
+    &:active:focus{
+      & > span{
+        filter: brightness(85%);
+        color: ${btnPrimaryColor};
+        background-color: ${btnPrimaryActiveBg};
+      }
+    }
+
+    &:active{
+      & > span{
+        filter: brightness(85%);
+        color: ${btnPrimaryColor};
+        background-color: ${btnPrimaryActiveBg};
+      }
+    }
+  `}
+
+  ${({ circle, ghost, color }) => circle && ghost && color === 'default' && `
+    & > span{
+      color: ${btnDefaultGhostColor};
+    }
+
+    &:focus:not([aria-disabled="true"]){
+      & > span{
+        background-color: ${btnDefaultGhostBg} !important;
+        background-clip: padding-box;
+        box-shadow: 0 0 0 ${borderSize} ${btnDefaultGhostBorder};
+        border-color: transparent !important;
+      }
+    }
+
+    &:hover{
+      & > span{
+        filter: brightness(90%);
+        background-color: ${btnDefaultGhostBg} !important;
+      }
+    }
+
+    &:active:focus{
+      & > span{
+        filter: brightness(85%);
+        background-color: ${btnDefaultGhostActiveBg};
```

Comment:
```suggestion
        background-color: ${btnDefaultGhostActiveBg} !important;
```

Otherwise, the `background-color` prop of the previous `:focus` block will take precedence over this.